src/server/ApiRoutes/Hold.js

```diff
@@ -118,13 +118,15 @@ function getDeliveryLocations(barcode, patronId, accessToken, cb, errorCb) {
     }
   )
   .then(barcodeAPIresponse => {
-    const deliveryLocationWithAddress = mapLocationDetails(
-      barcodeAPIresponse.data.itemListElement[0].deliveryLocation
-    );
+    const eddRequestable = (barcodeAPIresponse.data.itemListElement[0].eddRequestable) ?
```

Comment:
I believe they designed this endpoint to take multiple items!
So the endpoint could be 

`https://api.nypltech.org/api/v0.1/request/deliveryLocationsByBarcode?barcodes[]=33433035882855,33433035882856,33433035882857&patronId=6367028`

actually! I can try to test it.